administrator/components/com_media/resources/scripts/components/browser/items/item.js

```diff
@@ -16,7 +16,7 @@ export default {
          * Return the correct item type component
          */
         function itemType() {
-            let imageExtensions = ['jpg', 'png', 'gif'];
+            let imageExtensions = ['jpg', 'jpeg', 'png', 'gif'];
```

Comment:
This needs to be refactored to be flexible and extendable, but it is in some kind out of the scope of this PR.